data/gui/window/mp_match_history.cfg

```diff
@@ -1,5 +1,40 @@
 #textdomain wesnoth-lib
 
+#define _GUI_HORIZONTAL_TAB ID
+	[row]
+
+		[column]
+			horizontal_grow = true
+			vertical_alignment = top
+
+			[grid]
+
+                [row]
+                    grow_factor = 0
+
+                    [column]
+                        grow_factor = 0
+
+                        border = "all"
+                        border_size = 5
+                        horizontal_alignment = "left"
+
+                        [label]
+							id = "{ID}_game_name"
+                            definition = "default"
+                        [/label]
+
+                    [/column]
+
+                [/row]
```

Comment:
There's a mix of tabs and spaces in this file.

Reply:
Wasn't the whitespace fixup script changed to be able to catch this?

Reply:
It was for Lua files, I expected wmlindent to cover .cfg files. Forgot that wmlindent isn't run on the GUI files...

Reply:
I'm looking at how many changes it would need to get the GUI files consistent (just on a per-file basis, using either one or the other). You'll note that those files aren't part of #7424, I'm still looking at them...